package-system/pyside2/Findpyside2.cmake

```
@@ -123,11 +123,16 @@ set_target_properties(${MY_NAME}::ShibokenTool PROPERTIES IMPORTED_LOCATION "${$
 add_executable(${TARGET_WITH_NAMESPACE}::ShibokenTool ALIAS ${MY_NAME}::ShibokenTool)
 
 function(add_shiboken_project)
-    set(oneValueArgs NAMESPACE NAME WRAPPED_HEADER TYPESYSTEM_FILE GENERATED_FILES LICENSE_HEADER)
+    set(oneValueArgs MODULE_NAME NAMESPACE NAME WRAPPED_HEADER TYPESYSTEM_FILE GENERATED_FILES LICENSE_HEADER)
     set(multiValueArgs INCLUDE_DIRS DEPENDENCIES)
     cmake_parse_arguments(add_shiboken_project "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN} )
     
     # Validate arguments
+    if (NOT add_shiboken_project_MODULE_NAME)
+        message("You must provide a module name matching the package name in the xml typesystem file.")
+        message(FATAL_ERROR "This is the name of the output Python module.")
```

Combine the FATAL_ERROR into a single sentence
```suggestion
        message(FATAL_ERROR "You must provide a module name matching the package name in the xml typesystem file. This is the name of the output Python module.")
```